### PR TITLE
Adds alerting plugin related docs

### LINKED ISSUE #207: [FEATURE] Add documentation for plugin APIs in GETTING_STARTED and Sphinx docs
### Is your feature request related to a problem?
With new alerting plugin APIs in this [PR](https://github.com/opensearch-project/opensearch-py/pull/93). We'd like to add documentation to the sphinx docs and add examples on how to use the APIs in the getting started guide.

### What solution would you like?
Similar to other examples and class based docs in the sphinx docs.

### PR DESCRIPTION
**Pull request by harshavamsi**
Signed-off-by: Harsha Vamsi Kalluri <harshavamsi096@gmail.com>

### Description
Adds documentation and examples for newly introduced alerting plugin APIs in #93 

### Issues Resolved
Closes #207 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by axeoman — APPROVED** [review]
LGTM, thanks :+1:

**Review by dblock — APPROVED** [review]

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+    - [Deleting a document](#deleting-a-document)
+    - [Deleting an index](#deleting-an-index)
+  - [Using plugins](#using-plugins)
+    - [Alerting plugin](#alerting-plugin)
+      - [**Searching for monitors**](#searching-for-monitors)
+      - [**Getting a monitor**](#getting-a-monitor)
+      - [**Creating a monitor**](#creating-a-monitor)
+      - [**Creating a destination**](#creating-a-destination)
+      - [**Getting alerts**](#getting-alerts)
+      - [**Acknowledge alerts**](#acknowledge-alerts)
```
Comment (dblock) [inline GETTING_STARTED.md:19]: Why are these **bold**? :)

Comment (harshavamsi) [reply GETTING_STARTED.md:19]: Everything looked the same on the page. I felt that we would need some separation between the main headings and the sub headings.

**Review by VachaShah — APPROVED** [review]

File: GETTING_STARTED.md (hunk 1 above)
Comment (dblock) [reply GETTING_STARTED.md:19]: 🤔 that's very un-markdown

**Pull request merged**